tests/components/keenetic_ndms2/test_config_flow.py

```diff
@@ -167,3 +168,71 @@ async def test_connection_error(hass, connect_error):
     )
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["errors"] == {"base": "cannot_connect"}
+
+
+async def test_ssdp_works(hass: HomeAssistantType, connect):
+    """Test host already configured and discovered."""
+
+    discovery_info = MOCK_SSDP_DISCOVERY_INFO.copy()
+    result = await hass.config_entries.flow.async_init(
+        keenetic.DOMAIN,
+        context={CONF_SOURCE: config_entries.SOURCE_SSDP},
+        data=discovery_info,
+    )
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "user"
```

Comment:
Complete the configuration flow, to ensure there are no side effects created because it was initiated by the discovery.